docs/guide/search/solr.md

```diff
@@ -4,15 +4,15 @@
 
 ## How to set up Solr search engine
 
-!!! note "Enable the bundle"
+!!! note "Installing the bundle"
 
-    If you have previously disabled the bundle, add/update composer dependencies:
+    If you have previously removed the bundle, add/update composer dependencies:
     ``` bash
-    composer require --no-update ezsystems/ezplatform-solr-search-engine:~2.0
+    composer require --no-update ezsystems/ezplatform-solr-search-engine:~3.0
     composer update
     ```
-
-    Make sure `EzPublishSolrSearchEngineBundle` is activated with the following line in the `app/AppKernel.php` file: `new EzSystems\EzPlatformSolrSearchEngineBundle\EzSystemsEzPlatformSolrSearchEngineBundle()`
+    
+    Symfony Flex should enable the bundle for you when installing the package.
```

Comment:
```suggestion
    Symfony Flex should enable the bundle for you when installing the package.
```
"Should enable" or "will enable"? 

If "should", I would mention the circumstances that can prevent Symfony Flex from enabling the bundle, and if/how one can enable Solr manually.

Reply:
I didn't have the time to test and verify so that is why I as unsure. If anyone has install, remove solr, and try to require it to see what flex does. Or we omit the omit should, and take it as a bug if it turns out to be wrong 😉

Reply:
```suggestion
    Symfony Flex will enable the bundle for you when installing the package.
```